scripts/generate-locales.ts

```diff
@@ -111,6 +114,136 @@ function escapeField(parent: string, module: string): string {
+
+  // Initialize stored weekdays
+  let storedWeekdays: { wide: string[]; abbr: string[] } = {
+    wide: [],
+    abbr: [],
+  };
+
+  // Import the current weekday values
+  try {
+    const imported = await import(`file:${weekdayPath}`);
+    storedWeekdays = imported.default;
+  } catch {
+    console.log(`Creating weekday.ts file for locale ${locale}.`);
+  }
+
+  // Generate correct weekday values
+  const wide: string[] = [];
+  const abbr: string[] = [];
+  // eslint-disable-next-line no-restricted-globals
+  const intlWeekdayLong = new Intl.DateTimeFormat(locale, { weekday: 'long' });
+  // eslint-disable-next-line no-restricted-globals
+  const intlWeekdayShort = new Intl.DateTimeFormat(locale, {
+    weekday: 'short',
+  });
+  for (let i = 0; i < 7; i++) {
+    const date: Date = new Date(2020, 0, i + 4);
+    // January 4-10, 2020 are Sunday to Saturday
+    wide.push(intlWeekdayLong.format(date));
+    abbr.push(intlWeekdayShort.format(date));
+  }
+
+  // Update stored weekdays
+  storedWeekdays.wide = wide;
+  storedWeekdays.abbr = abbr;
+
+  // Normalize stored weekdays before saving
+  const normalizedWeekdays = normalizeDataRecursive(storedWeekdays);
+
+  // Write updated values back to the file
+  const updatedContent = `${autoGeneratedCommentHeader}
+export default ${JSON.stringify(normalizedWeekdays, null, 2)};`;
+  await writeFile(weekdayPath, await formatTypescript(updatedContent));
+}
+
+// Function to update month values for a given locale
+async function updateMonthForLocale(locale: string): Promise<void> {
+  if (!isValidLocale(locale)) return;
+
+  const dateFolderPath: string = resolve(pathLocales, locale, 'date');
+  const monthPath: string = resolve(dateFolderPath, 'month.ts');
+
+  // Initialize stored months
+  let storedMonths: { wide: string[]; abbr: string[] } = { wide: [], abbr: [] };
+
+  // Import the current month values
+  try {
+    const imported = await import(`file:${monthPath}`);
+    storedMonths = imported.default;
+  } catch {
+    console.log(`Creating month.ts file for locale ${locale}.`);
+  }
+
+  // Generate correct month values
+  const wide: string[] = [];
+  const abbr: string[] = [];
+  // eslint-disable-next-line no-restricted-globals
+  const intlMonthLong = new Intl.DateTimeFormat(locale, { month: 'long' });
+  // eslint-disable-next-line no-restricted-globals
+  const intlMonthShort = new Intl.DateTimeFormat(locale, { month: 'short' });
+  for (let i = 0; i < 12; i++) {
+    const date: Date = new Date(2020, i, 1);
+    wide.push(intlMonthLong.format(date));
+    abbr.push(intlMonthShort.format(date));
+  }
+
+  // Update stored months
+  storedMonths.wide = wide;
+  storedMonths.abbr = abbr;
+
+  // Normalize stored months before saving
+  const normalizedMonths = normalizeDataRecursive(storedMonths);
+
+  // Write updated values back to the file
+  const updatedContent = `${autoGeneratedCommentHeader}
+export default ${JSON.stringify(normalizedMonths, null, 2)};`;
```

Comment:
The following hopefully fixes the code highlighting in the browser.

```suggestion
  const updatedContent = `
    ${autoGeneratedCommentHeader}
    export default ${JSON.stringify(normalizedMonths, null, 2)};
    `;
```